che-plugins.yaml

```diff
@@ -36,7 +36,7 @@ plugins:
         image: 'quay.io/eclipse/che-buildah-base:1.14.0-b0c91d3'
         mountSources: true
         memoryLimit: 1Gi
-  - id: eclipse/che-async-pv-plugin/nightly
+  - id: eclipse/che-async-pv-plugin/next
```

Comment:
This is a plugin version, I don't think it should be changed here

Reply:
alright, keeping nightly